Documentation/Internals/MakefileSupport.md

```diff
@@ -0,0 +1,29 @@
+# Makefile Generation Support
+
+SwiftPM has support for generating makefiles for simple Swift packages.  Some of
+the advanced features like modulemap generation, package dependencies, etc are
+out-of-scope unless required in future. The makefile support is accessible using
+an hidden flag and is intended to be used only for Swift.org related projects for
+bootstrapping them on the Swift CI.
+
+To generate makefile for a package, run the following command:
+
+```sh
+$ swift package __internal-tools --generate-makefile
+```
+
+**Note**: This command is hidden from the help.
+
+SwiftPM will generate files inside the directory `swift-ci/`.  The generated
```

Comment:
This seems like a funny name to me, something w.r.t. "bootstrap" would be more obvious to me. (swift-ci is the use case, not the mechanism)

Reply:
Makes sense. I think we can just use `bootstrap/` unless you have a better suggestion.